setup.py

```diff
@@ -19,17 +19,17 @@
 here = path.abspath(path.dirname(__file__))
 
 install_requires = [
-    'apispec>=1.1.0',
+    'apispec==2.0.2',
     'multidict',
     'pyyaml',
 ]
 marshmallow_require = [
-    'marshmallow<3.0.0rc1',
-    'apispec_marshmallow_advanced>=0.3',
+    'marshmallow==2.21.0',
+    'apispec_marshmallow_advanced==0.4',
 ]
 serpyco_require = [
-    'apispec_serpyco>=0.15',
-    'serpyco>=0.17',
+    'apispec_serpyco==0.17',
+    'serpyco==0.17.4',
```

Comment:
In theory `setup.py` should list versions we are compatible for but not set exact versions (if we are compatible with several) as this can cause problems with other packages having the same dependencies.

Reply:
I do agree but we probably need to find a best way to find those case as the current approach lead to broken develop. This dependencies issue are complex to troubleshoot.  
i suggest we later have a similar approach than for tracim, with pip-tools that generate requirement.txt that does work and we test travis on requirement.txt.
So, i suggest to be strict now to solve the issue rapidly and then we do check to find a better approach.

Reply:
create a issue about this here : #202